catalogue/webapp/pages/search/index.tsx

```diff
@@ -133,7 +133,15 @@ export const SearchPage: NextPageWithLayout<Props> = ({
                 <div className="container">
                   <SectionTitle sectionName="Stories" />
 
-                  <StoriesGrid stories={stories} />
+                  <StoriesGrid
+                    stories={stories}
+                    dynamicImageSizes={{
+                      xlarge: 1 / 5,
+                      large: 1 / 5,
+                      medium: 1 / 2,
+                      small: 1 / 2,
```

Comment:
don't cards take up more than half the screen width on mobile?

Reply:
On the overview page I left them 2 per row as I thought "it's an overview and there's more content to look at below", but I think that's something I should defo review with @DominiqueMarshall when she's back and we go through Search!